src/main/java/org/mitre/synthea/world/agents/Provider.java

```diff
@@ -61,6 +62,9 @@
       Config.get("generate.providers.selection_behavior", "nearest").toLowerCase();
   private static IProviderFinder providerFinder = buildProviderFinder();
 
+  public static final boolean appendNumbersToNames =
+      Config.getAsBoolean("generate.append_numbers_to_person_names", false);
+  
```

Comment:
This should probably just be inside the new `Names` class.

Reply:
Every instance of calling `fakeFirstName` or `fakeLastName` also checked this boolean immediately after, so I moved this and the subsequent check and adding numbers into those 2 methods.